openstack-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1004 @@
+              'hostname 2': true,
+              'hostname 3': true,
+              'hostname 4': true,
+              'hostname 5': true,
+              'hostname 6': true,
+              job: true,
+              'job 1': true,
+              'job 2': true,
+              'job 3': true,
+              'job 4': true,
+              'job 5': true,
+              'job 6': true,
+              'resourcetype 1': true,
+              'resourcetype 2': true,
+              'resourcetype 3': true,
+              'resourcetype 4': true,
+              'resourcetype 5': true,
+              'resourcetype 6': true,
+            },
+            indexByName: {
+              instance: 0,
+              '__name__ 1': 1,
+              'resourcetype 1': 2,
+              'Value #A': 3,
+              'Value #B': 4,
+              'Value #C': 5,
+              'Value #D': 6,
+              'Value #E': 7,
+              'Value #F': 8,
+              'Time 2': 9,
+              '__name__ 2': 10,
+              'hostname 2': 11,
+              'job 2': 12,
+              'resourcetype 2': 13,
+              'Time 3': 14,
+              '__name__ 3': 15,
+              'hostname 3': 16,
+              'job 3': 17,
+              'resourcetype 3': 18,
+              'Time 4': 19,
+              '__name__ 4': 20,
+              'hostname 4': 21,
+              'job 4': 22,
+              'resourcetype 4': 23,
+              'Time 5': 24,
+              '__name__ 5': 25,
+              'hostname 5': 26,
+              'job 5': 27,
+              'resourcetype 5': 28,
+              'Time 6': 29,
+              '__name__ 6': 30,
+              'hostname 6': 31,
+              'job 6': 32,
+              'resourcetype 6': 33,
+              'Time 1': 34,
+              'hostname 1': 35,
+              'job 1': 36,
+            },
+            renameByName: {
+              Time: '',
+              'Value #A': 'Disk available',
+              'Value #B': 'Disk used',
+              'Value #C': 'Memory avaliable',
+              'Value #D': 'Memory used',
+              'Value #E': 'VCPUs available',
+              'Value #F': 'VCPUs used',
+              hostname: 'Hostname',
+              instance: 'Instance',
+              openstack_placement_resource_total: 'Total',
+              openstack_placement_resource_usage: 'In use',
+              resourcetype: 'Resource',
+            },
+            includeByName: {},
+          },
+        },
+      ]),
+
+    domains:
+      commonlib.panels.generic.stat.base.new(
+        'Domains',
+        targets=[t.domains],
+        description='The number of domains for the OpenStack cloud.',
+      )
+      + stat.options.withGraphMode('none'),
+
+    projects:
+      commonlib.panels.generic.stat.base.new(
+        'Projects',
+        targets=[t.projects],
+        description='The number of projects for the OpenStack cloud.',
+      )
+      + stat.options.withGraphMode('none'),
+
+    regions:
```

Comment:
instead of using a generic stat base new, you can switch this to a stat info. This will remove the color since they aren't representing anything and align a bit more with the common lib/re-usability paradigm